src/mon/OSDMonitor.cc

```diff
@@ -7354,7 +7354,7 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     string field;
     cmd_getval(g_ceph_context, cmdmap, "field", field);
     if (field != "max_objects" && field != "max_bytes") {
-      ss << "unrecognized field '" << field << "'; max_bytes of max_objects";
+      ss << "unrecognized field '" << field << "'; not in max_bytes or max_objects";
```

Comment:
imho, 

```should be "max_bytes" or "max_objects"```

would be easier for user to understand?

"not in max_bytes or max_objects" sounds like a python statement to me =)